core/controller/hook/github.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * core/controller/hook/github.js
+ * Handles all GitHub inputs
+ *
+ * @exports {Object} default - Koa router
+ */
+
+import Router from 'koa-router'
+
+import { Config, Log } from '~/app'
+
+// TODO: GitHub hook integration
+let route = new Router({
```

Comment:
After using it a while, I really enjoy how simple it is to lay down requirements for different paths. For instance, being able to make the GitHub hook only respond to this path, automatically verify the key, and then pass along a database object to the next route. Best part is that I have yet to get into some weird circumstance where it doesn't act how I want it to.